mackerel/api.go

```diff
@@ -73,13 +73,18 @@ func (api *API) urlFor(path string, query string) *url.URL {
 	return newURL
 }
 
+func (api *API) getUA() string {
+	if api.UA != "" {
+		return api.UA
+	}
+	return "mackerel-agent/0.0.0"
+}
+
 var apiRequestTimeout = 30 * time.Second
 
 func (api *API) do(req *http.Request) (resp *http.Response, err error) {
 	req.Header.Add("X-Api-Key", api.APIKey)
-	req.Header.Add("X-Agent-Version", version.VERSION)
```

Comment:
We don't use `X-Agent-Version` and `X-Revision` header currently, so maybe it's okay. maybe……